mat/matrix.go

```diff
@@ -302,63 +302,26 @@ func Cond(a Matrix, norm float64) float64 {
 	case math.Inf(1):
 		lnorm = lapack.MaxRowSum
 	}
+
+	tmp := getWorkspace(m, n, false)
```

Comment:
I don't think we need this now. The factorization types already guard against modifying the matrix.